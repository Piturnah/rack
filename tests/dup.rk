10 dup print print