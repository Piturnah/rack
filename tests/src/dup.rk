fn main in 
  10 dup print print 
  0xffffffffffffffff dup print print
end
