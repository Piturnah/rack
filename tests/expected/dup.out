----STDOUT----
10
10
18446744073709551615
18446744073709551615

----STDERR----
